### ISSUE TITLE
advanced-save-file

### ISSUE THREAD

**Issue by trusktr** [Osmose/advanced-open-file#119]
A reciprocal advanced-save-file would be awesome!

**Comment by Osmose** [comment]
You mean for saving a file opened in a new tab?

**Comment by trusktr** [comment]
@Osmose Yeah, or like when pressing ctrl-shift-s for Save As, it opens the OS' file explorer outside of Atom, but I'd much rather have an auto-completing palette inside of Atom similar to the advanced-open-file palette. Plus, it would be the same across OSes since it is inside Atom UI, not the OS.
